test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1

```diff
@@ -299,3 +299,24 @@ Describe "Connection" -Tag "CI", "RequireAdminOnWindows" {
         Test-Connection $UnreachableAddress -TcpPort 80 -TimeOut 1 | Should -BeFalse
     }
 }
+
+Describe "Test-Connection should run in the default synchronization context (threadpool)" -Tag "CI" {
+    It "Test-Connection works after constructing a WindowsForm object" -Skip:(!$IsWindows) {
+        $pwsh = Join-Path $PSHOME "pwsh"
+        $pingResults = & $pwsh -NoProfile {
```

Comment:
Why not add -NoInteractive and don't load PSReadline?

Reply:
`PSReadLine` won't be loaded in this case. It's essentially just running a command using `pwsh -EncodedCommand`.